### PR TITLE
Homework2

### PR DESCRIPTION
**Pull request by alicezangliger**

### CODE REVIEW

File: Homework2/HW2Tests/HWorkTests.cs [hunk 1]
```diff
@@ -0,0 +1,101 @@
+using Homework2;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace HW2Tests
+{
+    [TestClass]
+    public class HWorkTests
+    {
+        [TestMethod]
```
Comment (IgorKheidorov) [inline Homework2/HW2Tests/HWorkTests.cs:9]: Add to test name Negative or positive depending on test type

File: Homework2/HW2Tests/HWorkTests.cs [hunk 2]
```diff
@@ -0,0 +1,101 @@
+using Homework2;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace HW2Tests
+{
```
Comment (IgorKheidorov) [inline Homework2/HW2Tests/HWorkTests.cs:5]: No checks for null string

File: Homework2/Homework2/SymbolString.cs [hunk 3]
```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace Homework2
+{
```
Comment (IgorKheidorov) [inline Homework2/Homework2/SymbolString.cs:3]: Add comments to classes and methods

File: Homework2/Homework2/SymbolString.cs [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+                    else
+                    {
+                        count = 1;
+                    }
+                }
+
+            }
+            return maxNumb;
+        }
+
```
Comment (IgorKheidorov) [inline Homework2/Homework2/SymbolString.cs:58]: Think how to combine method EqualNumbers and EqualLatin with common functionality? they do almost the same. I propose to use special method for Check IsLatin or Digit inside

**Review by IgorKheidorov — APPROVED** [review]
Not bad, the main issue not add binary files to commit, only *.cs , *.proj. *.sln has to be in commit
